src/controllers/requests.js

```diff
@@ -93,5 +101,56 @@ class RequestsController {
       requests
     });
   }
+
+  static async updateOpenRequests(req, res) {
+    const { user } = req;
+
+    if (!req.body) {
+      return res.status(400).json({
+        error: res.__(emptyUpdate)
+      });
+    }
+
+    const request = await Request.findOne({
+      where: { id: req.params.id }
+    });
+
+    if (!request) {
+      return res.status(400).json({
+        error: res.__(notExistRequest)
+      });
+    }
+
+    if (request.userId !== user.id) {
+      return res.status(401).json({
+        error: res.__(notYourRequest)
+      });
```

Comment:
Could this be put into the middleware, let's keep only the actual business into the controller